client/actions/assignments/notifications.ts

```diff
@@ -315,13 +315,14 @@ const onAssignmentDeleteFailed = (_e, data) => (
         if (get(data, 'items.length', 0) > 0 &&
                 get(data, 'user') === currentUserId &&
                 get(data, 'session') === sessionId) {
-            const msg = data.items.map((i) => gettext('Delete of {{ type }} assignment \'{{ slugline }}\' failed',
+            const msg = data.items.map((i) => gettext(
```

Comment:
Sometimes the error may be different then `state == 'in_progress'`, such as: https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/planning/planning.py#L1009

Maybe we should handle the errors better here:
https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/planning/planning.py#L1009